akka-docs/src/main/paradox/scala/stream/stream-rate.md

```diff
@@ -89,11 +90,11 @@ Java
 :   @@snip [StreamBuffersRateDocTest.java]($code$/java/jdocs/stream/StreamBuffersRateDocTest.java) { #buffering-abstraction-leak }
 
 Running the above example one would expect the number *3* to be printed in every 3 seconds (the `conflateWithSeed`
```

Comment:
`conflateWithSeed` is defined in `Source`, `Flow` and `SubFlow`. So I didn't insert any link to this here.